src/components/CalcPage.js

```diff
@@ -0,0 +1,15 @@
+import Calculator from './Calculator';
+import Header from './Header';
+import './styles/Calculator.css';
+
+const CalcPage = () => (
+  <div>
+    <Header />
+    <div className="calculator-container">
+      <h2>Let&apos;s do some math!</h2>
+      <Calculator />
```

Comment:
- [ ] The calculator doesn't work. I recommend fixing that so that your project is complete.

Reply:
Hi @medaminedev66, In which way the calculator doesn't work? 
Could you please provide extra feedback?

[Loom video doing basic operations with calculator](https://www.loom.com/share/219fbf5ea515418b85b178b19b19dd5c)